testing/logging/test_reporting.py

```diff
@@ -1152,3 +1152,82 @@ def test_bad_log(monkeypatch):
     )
     result = testdir.runpytest()
     result.assert_outcomes(passed=1)
+
+
+def test_log_file_is_in_pytest_ini_rootdir(testdir):
```

Comment:
Please add docstrings to each test referencing #7352, so it is easier to find the issue that lead to the creation of the test later. 👍

Reply:
thanks, i assume a typo of #7350 - will update

Reply:
oops yes 😁